server/tournaments/index.ts

```diff
@@ -1699,6 +1738,33 @@ const chatCommands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
+			if (room.lastTournamentData && room.lastTournamentData.type === 'tree') {
+				let tourString = '';
+				let tourData = [];
+				const tourWinner = room.lastTournamentData.rootNode.team;
+				let roundCounter = Math.ceil(Math.log(room.lastTournamentData.playersLength) / Math.log(2));
+				tourData.push(room.lastTournamentData.rootNode);
+				while (moreRound(tourData)) {
```

Comment:
Actually, `moreRound` in general is unnecessary here.

```suggestion
				while (tourData.some(node => node.children)) {
```

Reply:
You might need a `!!` cast, if TypeScript complains about `some`'s return type.